@AH-64D Apache Longbow/Addons/fza_ah64_us/model.cfg

```diff
@@ -1488,8 +1488,8 @@ class cfgModels
                 memory=1;
                 minValue=0;
                 maxValue=1;
-                angle0="rad +30";
-                angle1="rad -30";
+                angle0="rad -30";
```

Comment:
I need to add a left and right click zone. When held to the left, you get ENG1 FIRE, APU FIRE, ENG2 FIRE, AFT DECK FIRE lights and audios, when held to the right you get ENG 1 FIRE, APU FIRE, ENG 2 FIRE, AFT DECK FIRE lights and audios as well as the 2 discharge lights. When the switch is released it spring loads back to center.